packages/edit-site/src/components/template-actions/style.scss

```diff
@@ -0,0 +1,4 @@
+// The rename dropdown popover is open at the same time as the rename modal. The latter has to be higher.
+.edit-site-list__rename-modal {
+	z-index: z-index(".components-popover") + 1;
```

Comment:
Instead of calculating, should we add it to the list z-indexes that are meant to be `1` greater than the popover?

https://github.com/WordPress/gutenberg/blob/938e74777f6a16b1278f5e75872cfa20cbb8bd92/packages/base-styles/_z-index.scss#L122-L128

Reply:
Thanks. I realized that and just used `z-index(".components-popover")` as it was in [ca5ecf1](https://github.com/WordPress/gutenberg/pull/54277/commits/ca5ecf144a94ab3d574cee611683e5ee526e7432)

The document tree order takes care of the rest.

Or do you think we should add 1 as above to be safe?

Reply:
Hmmm, seeing as it's been done before

https://github.com/WordPress/gutenberg/blob/938e74777f6a16b1278f5e75872cfa20cbb8bd92/packages/base-styles/_z-index.scss#L122-L128

I'll add an entry. Thanks for finding that! 🙇🏻

Reply:
Done. Thanks @andrewserong !